src/main/java/edu/harvard/iq/dataverse/api/SavedSearches.java

```diff
@@ -173,16 +175,18 @@ public Response add(JsonObject body) {
 
     @DELETE
     @Path("{id}")
-    public Response delete(@PathParam("id") long doomedId) {
-        boolean disabled = true;
-        if (disabled) {
-            return error(BAD_REQUEST, "Saved Searches can not safely be deleted because links can not safely be deleted. See https://github.com/IQSS/dataverse/issues/1364 for details.");
-        }
+    public Response delete(@PathParam("id") long doomedId, @QueryParam("unlink") boolean unlink) {
         SavedSearch doomed = savedSearchSvc.find(doomedId);
         if (doomed == null) {
             return error(NOT_FOUND, "Could not find saved search id " + doomedId);
         }
-        boolean wasDeleted = savedSearchSvc.delete(doomedId);
+        boolean wasDeleted;
+        try {
+            wasDeleted = savedSearchSvc.delete(doomedId, unlink);
```

Comment:
Do we want to put this in a command? There's a "CreateSavedSearchCommand"

Reply:
> Do we want to put this in a command? There's a "CreateSavedSearchCommand"

Hey Sekmiller, if my understanding is correct `Command` are used to share the same code between UI and API endpoints.

I note that `CreateSavedSearchCommand` is not used anymore (it was used in `DataversePage.java` for UI purposes in the past).
I note that adding Saved Search using POST API is not using `CreateSavedSearchCommand`.
I note that the code has been initiated back in 2015 and has been for a long time labelled experimental.

It could be prematured to create a `Command`  that could be called from interface. What do you think ?
Sorry, but I'm more "quick win and no refactor while it's not needed" kind of mind.

Reply:
One reason that I suggested the command was because commands also check for permissions. I didn't notice that this is a superuser only feature - so that permission check wouldn't be available in the command. 
We can leave it be for now, but if we ever open it up to other users we might want to revisit the command issue.